Examples.xcodeproj/project.pbxproj

```diff
@@ -1361,6 +1370,8 @@
 				3E44BA55203B54120072DE4B /* MultipleImagesExample.swift in Sources */,
 				3E1806101EAA800A004DB131 /* UserLocationAnnotationExample.m in Sources */,
 				961962911C581700002D3DAB /* main.m in Sources */,
+				6FAC014722718FCC00F7FA18 /* OrnamentsLayoutExample.swift in Sources */,
+				05FA53A21FE2FA1E001F3D7D /* PolygonAnnotationExample.m in Sources */,
```

Comment:
Looks like this example snuck back in (removed in https://github.com/mapbox/ios-sdk-examples/pull/352). Once this is removed, LGTM~

Reply:
This is a weird one. Not sure why it came back. Rebasing.